src/components/app-configure/SettingCard.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts" setup>
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VSeperator from '@/components/lib/VSeperator/VSeperator.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+</script>
+
+<template>
+  <VCard variant="elevated" class="settings-card">
+    <VStack direction="column" gap="2rem" class="flex-grow">
+      <h3 class="text-uppercase">
+        <slot name="title" />
+      </h3>
+      <VSeperator class="settings-card-separator" />
+      <div class="body-1 settings-card-description">
+        <slot name="description" />
+      </div>
+      <slot />
+    </VStack>
+  </VCard>
+</template>
+
+<style scoped>
+.settings-card {
+  padding: 2rem 1.5rem;
+}
+
+.settings-card-separator {
+  width: calc(100% + 3rem);
+  margin: 0 -1.5rem;
```

Comment:
Just a note for now. See if there's a way to make this independent of the card padding.